src/types/dto/attachments.dto.ts

```diff
@@ -0,0 +1,25 @@
+import { boolean, z } from 'zod'
+
+export const CreateAttachmentRequestSchema = z.object({
+  taskId: z.string().uuid(),
+  filePath: z.string(),
+  fileSize: z.number(),
+  fileType: z.string(),
```

Comment:
It might be better to make fileType an enum that only accepts a restricted list of file types. E.g. we don't want users uploading `exe` files.

We can include common document types like pdf, etc, media like jpeg, png, etc - basically anything we have a corresponding icon for